### PR TITLE
[openwebnet] Fixes #8524 and #8547, updated openwebnet4j lib to 0.3.1. Updated README

### LINKED ISSUE #8524: [openwebnet] BUS gateway goes offline and back online if no event is received for a while
## Expected Behavior
Connection to gateway remains active even if no event message is received for a while

## Current Behavior
If the binding does not receive any event message from BUS gateway for 2mins, then it disconnects and reconnects and the gateway Thing goes offline and then back online.

## Possible Solution
Check if the gateway is still reachable before disconnecting MON session.

## Steps to Reproduce (for Bugs)
Disconnection events are reported on the OH log.

## Your Environment
* Version used: 2.5.9

### LINKED ISSUE #8547: [openwebnet] Dimmers are rediscovered as switch even though already added for same BUS address
openwebnet binding v2.5.9

For dimmers when a discovery is run they should be powered on in order to be discovered as dimmers otherwise they are found as switch type. Even after correct a discovery the same dimmer can be found again as switch if they are not powered up on repeat discovery runs. 
Proposed solution would be to not discover the device again if a thing already exists with the same BUS address.

### PR DESCRIPTION
**Pull request by mvalla**
Fixes #8524 and fixes #8547, updated openwebnet4j lib to 0.3.1. Updated README

### CODE REVIEW

File: bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/handler/OpenWebNetBridgeHandler.java [hunk 1]
```diff
@@ -314,13 +314,27 @@ protected void registerDevice(String ownId, OpenWebNetThingHandler thingHandler)
...
+     *
+     * @param ownId the device OpenWebNet id
+     * @return the registered device Thing handler or null if the id cannot be found
+     */
+    public @Nullable OpenWebNetThingHandler getRegisteredDevice(String ownId) {
+        if (registeredDevices.containsKey(ownId)) {
+            return registeredDevices.get(ownId);
+        } else {
+            return null;
         }
```
Comment (fwolter) [inline bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/handler/OpenWebNetBridgeHandler.java:338]: `Map.get()` returns null if the key doesn't exist. You could simply do:
```suggestion
        return registeredDevices.get(ownId);
```

**Review by fwolter — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/handler/OpenWebNetBridgeHandler.java (hunk 1 above)
Comment (mvalla) [reply bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/handler/OpenWebNetBridgeHandler.java]: fixed. 
Could you also set this PR label (bug) so it gets tracked in the changelog? (apparently I cannot set label to my PR request), thanks.

**Comment by fwolter** [comment]
Forgot to approve it, LGTM

**Comment by mvalla** [comment]
Thanks @fwolter for merging. I see you tagged it **oh25x**. Does it mean that the PR will be integrated automatically to the 2.5.x branch as well, or should I do something?

**Comment by fwolter** [comment]
I cherry-picked it to 2.5.x. See https://github.com/openhab/openhab-addons/commits/2.5.x

**Pull request merged**